tests/test_project.py

```diff
@@ -545,6 +849,35 @@ def helper_test_add_strain_maps(project):
         except Exception as e:
             assert type(e) == SherlockAddStrainMapsError
 
+        # happy path test missing because needs valid file
+        try:
+            strain_map = "Missing strain map.jpg"
+            strain_map_image_properties = (
+                BoardBounds([(1.0, 2.0), (3.0, 4.0), (1.0, 2.0), (1.0, 2.0)]),
+                "in",
+                ImageBounds(0.0, 0.0, 10.0, 8.0),
+                LegendBounds(1.0, 2.0, 4.0, 2.0),
+                StrainMapLegendOrientation.VERTICAL,
+                20.0,
+                50.0,
+                "µε",
+            )
+            project.add_strain_maps(
+                "Tutorial Project",
+                [
+                    (
+                        strain_map,
+                        "This is the strain map image for the project",
+                        StrainMapsFileType.IMAGE,
+                        strain_map_image_properties,
+                        ["Main Board"],
+                    )
+                ],
+            )
+            pytest.fail("No exception raised when using an invalid parameter")
+        except Exception as e:
+            assert type(e) == SherlockAddStrainMapsError
+
```

Comment:
I don't see any tests for type EXCEL. There should be at least one test for this to ensure that the code is using the same logic for CSV and EXCEL.